src/discolight/augmentations/imagecompression.py

```diff
@@ -0,0 +1,44 @@
+"""A Image compression augmentation."""
+import cv2
+from discolight.params.params import Params
+from .augmentation.types import ColorAugmentation, BoundedNumber
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class ImageCompression(ColorAugmentation):
+
+    """Image compress the given image.
+
+    Works for both jpeg/jpg and png format.
```

Comment:
Not sure if this line is necessary. All augmentations should work for any image that can be loaded with OpenCV.